auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -1418,21 +1418,25 @@ public TestContext(ProcessingEnvironment processingEnvironment, String packageNa
       this.properties = properties;
     }
 
-    public ProcessingEnvironment processingEnvironment() {
+    @Override public ProcessingEnvironment processingEnvironment() {
       return processingEnvironment;
     }
 
-    public String packageName() {
+    @Override public String packageName() {
       return packageName;
     }
 
-    public TypeElement autoValueClass() {
+    @Override public TypeElement autoValueClass() {
       return autoValueClass;
     }
 
-    public Map<String, ExecutableElement> properties() {
+    @Override public Map<String, ExecutableElement> properties() {
       return properties;
     }
+
+    @Override public Set<ExecutableElement> abstractMethods() {
+      return null;
```

Comment:
`emptySet()`? Seems like the code under test doesn't hit this anyway, but don't want future code to needlessly guard against NPEs